app/facades/quote_facade.rb

```diff
@@ -0,0 +1,15 @@
+class QuoteFacade
+  def self.top_10_query_quotes(search_query)
```

Comment:
Once you get the functionality working as expected, I'd try to refactor your facade to only have one method that gets you all the info you need. You are calling the same service method on lines 3 and 11 meaning you are making identical API calls. It would be more efficient if you could find a way to only make the API call one time.